test/sqllogictest/dates-times.slt

```diff
@@ -455,10 +455,10 @@ VALUES
 query T rowsort
 SELECT * FROM timestampwithtzish
 ----
-1999-12-31 06:00:00
-1999-12-31 07:11:01
-1999-12-31 13:46:01
-2000-01-01 08:00:00
+1999-12-31 01:00:00
+1999-12-31 02:11:01
+1999-12-31 08:46:01
+2000-01-01 03:00:00
```

Comment:
It seems weird that this has changed.

Reply:
I have seen this on my PRs as well. It seems that this particular sqllogictest query only works on buildkite and not locally.

Reply:
As discussed in slack, this indeed feels like a session timezone issue. In postgres:

> All timezone-aware dates and times are stored internally in UTC. They are converted to local time in the zone specified by the TimeZone configuration parameter before being displayed to the client.

Which means we need to configure the session (sqllogictest in this case) to use UTC.

Reply:
This works as expected for me (when I checkout this PR, running slt with `--rewrite-results` sets this file to what it is in `main`). Our session timezone parameter is hard coded to UTC and cannot be changed by the client.

My current idea is that because slt runs table stuff through postgres, the problem is related to the postgres server on your machines. You both run mac right? How are you running postgres? In a docker container or natively on the mac? Can you connect to your pg server and run `show timezone` and show the results here? If it doesn't have `UTC` in the result then I'm guessing that's related to the problem.